content/how-to/account-management/account-registration-code/index.md

```diff
@@ -1,26 +1,22 @@
 ---
 permalink: account-registration-code
 audit_date: '2019-01-22'
-title: Account registration code
+title: Account Registration Code
 type: article
 created_date: '2019-02-08'
 created_by: Rackspace Community
-last_modified_date: '2019-02-08'
-last_modified_by: Kate Dougherty
+last_modified_date: '2022-09-22'
+last_modified_by: Asmita Nakwa
 ---
 
-Rackspace uses an _account registration code_ to verify new users on an
-account, much like two-factor authentication. If an account administrator
-adds you as a new user, an account registration code is generated for
-your new user account and you receive an email notification that
-walks you through how to log in to the
-[Cloud Control Panel](https://login.rackspace.com)
-for the first time. Your first login is the only time that you need the
-registration code, unless you forget your user name.
+Rackspace uses an _account registration code_ to verify new users on an account, much like two-factor authentication as shown in below image. 
 
-The primary contact on the account is the only person who can change this
-code. When the primary contact adds a new user, the new user receives a
-registration code. The primary contact must give the new user their code.
+<img width="776" alt="accountregistrationcode1" src="/support/how-to/account-registration-code/accountregistrationcode1.png">
```

Comment:
fix alt tag to be a few words that describe what the image is showing

Reply:
Added "alphanumericcode"